apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -112,7 +112,6 @@ GoogleBlockly.Extensions.register('procedures_edit_button', function () {
     Blockly.useModalFunctionEditor &&
     this.inputList.length &&
     !this.workspace.isFlyout &&
-    this.workspace.id !== Blockly.functionEditor.getWorkspaceId() &&
     toolboxConfigurationSupportsEditButton(this)
```

Comment:
I can't comment on the actual line (109), but it looks like there's a TODO here that could use clean-up?

Reply:
Good catch, thanks!